student.rb

```diff
@@ -1,4 +1,6 @@
+require_relative 'high_five.rb'
 class Student
+  include High_five
```

Comment:
Good use of module! What you missed here is release 0.  The three classes have things in common.  You could do something like this:

```ruby
require_relative 'high_five.rb'

class Person
   include High_five

  attr_reader :age, :phase
  attr_accessor :name

  def initialize(options = {})
    @age = options.fetch(:age, 0)
    @name = options.fetch(:name, "")
  end
end
```